src/functions.sh

```diff
@@ -175,17 +175,16 @@ cmd_wrapper(){
     ERRLOG="sqm_error"
     if [ "$SILENT" -eq "1" ]; then
         ERRLOG="sqm_debug"
-        sqm_debug "cmd_wrapper: ${CALLERID}: invocation silenced by request, failure either expected or acceptable."
+        sqm_debug "cmd_wrapper: ${CALLERID}: invocation silenced by request, FAILURE either expected or acceptable."
         # The busybox shell doesn't understand the concept of an inline variable
         # only applying to a single command, so we need to reset SILENT
         # afterwards. Ugly, but it works...
         SILENT=0
     fi
 
-    sqm_trace "${CMD_BINARY} $@"
+    sqm_trace "cmd_wrapper: COMMAND: ${CMD_BINARY} $@"
     LAST_ERROR=$( ${CMD_BINARY} "$@" 2>&1 )
     RET=$?
-    sqm_trace "${LAST_ERROR}"
```

Comment:
Why drop this?

Reply:
Because we already do ` $ERRLOG "cmd_wrapper: ${CALLERID}: LAST ERROR: ${LAST_ERROR}"` a bit lower, and that also ends in the logfile. as is the `sqm_trace "${LAST_ERROR}"` basically either duplicated that information without an appropriate prefix or added an empty line in case of $RET -eq "0". So I believe this is not needed and likely never was.

Reply:
OK, fair enough